CIScripts/BuildAll.ps1

```diff
@@ -2,9 +2,49 @@
 . $PSScriptRoot\BuildFunctions.ps1
 . $PSScriptRoot\Job.ps1
 
+$ToolsAllowBranchOverride = $true
+
+# Additional logic for builds triggered from Gerrit
+if (Test-Path Env:GERRIT_CHANGE_ID)  {
```

Comment:
IMO if anything, we should specify repo name in param (like `contrail-sandesh`), and then build the URL (prepend) with github/gerrit and then CodiLime or Juniper